packages/commuter-client/src/reducers/index.js

```diff
@@ -6,6 +6,10 @@ const commuter = (state = {}, action) => {
     case types.REQUEST_CONTENTS:
       return Object.assign({}, state, { isFetching: action.isFetching });
     case types.RECEIVE_CONTENTS:
+    // filter out hidden files
+    action.entry.content = action.entry.content.filter((file) => {
+      return !file.name.startsWith('.')
```

Comment:
It's best if you don't mutate `content` in place here. There should be no side effects in the reducers (the action is changing in place here).

```js
const content = action.entry.type === 'directory' ?
    action.entry.content.filter((file) => {
      return !file.name.startsWith('.')
    }) : action.entry.content;
const entry = Object.assign({}, action.entry, { content });
return Object.assign({}, state, {
  entry,
  isFetching: action.isFetching,
...
```

However, to make this more readable I'd probably do a `let content` and then setup a switch statement to fill it in so this isn't nested in a long ternary.